src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -110,23 +125,26 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
             />
           )}
 
-          <TravellerSelection
-            setTravellerSelection={setTravellerSelection}
-            preassignedFareProduct={preassignedFareProduct}
-            selectableUserProfiles={selectableTravellers}
-            style={styles.selectionComponent}
-          />
+          {travellerSelectionMode !== 'none' && (
+            <TravellerSelection
+              setTravellerSelection={setTravellerSelection}
+              fareProductType={preassignedFareProduct.type}
+              travellerSelectionMode={travellerSelectionMode}
+              selectableUserProfiles={selectableTravellers}
+              style={styles.selectionComponent}
+            />
+          )}
 
-          <Zones
-            fromTariffZone={fromTariffZone}
-            toTariffZone={toTariffZone}
-            style={styles.selectionComponent}
-            isApplicableOnSingleZoneOnly={
-              preassignedFareProduct.isApplicableOnSingleZoneOnly
-            }
-          />
+          {zoneSelectionMode !== 'none' && (
+            <Zones
```

Comment:
Maybe call this `ZonesSelection` to make it more inline with the naming of the other selection component.